scroll-animations-1/Overview.bs

```diff
@@ -559,22 +587,20 @@ as follows:
 
     </div>
 
-    Note: Checking for {{endScrollOffset}} being the maximum scroll offset ensures that the common
+    Note: Checking for {{end}} being the maximum scroll offset ensures that the common
     case of a 'whole scroller' ScrollTimeline does not become inactive when you scroll to the end.
 
 1.  Return the result of evaluating the following expression:
 
     <blockquote>
-      <code>(<var>current scroll offset</var> - {{startScrollOffset}}) / ({{endScrollOffset}} - {{startScrollOffset}}) &times; [=effective time range=]</code>
+      <code>(<var>current scroll offset</var> - {{start}}) / ({{end}} - {{start}}) &times; [=effective time range=]</code>
     </blockquote>
 
-</div>  <!-- link-for-hint="ScrollTimeline" -->
-
-
 ## The 'animation-timeline' property ## {#animation-timeline}
```

Comment:
Perhaps not for this PR, but I would expect animation-timeline specification to live in CSS-ANIMATIONS spec ([css-animations-2](https://drafts.csswg.org/css-animations-2/)?)

Reply:
I am not sure if that is necessary. Unless we decide to move Scroll Timeline into web-animations as well. Otherwise keeping the css syntax and its JS equivalent next to each other makes more sense to me.